src/index.html

```diff
@@ -7,8 +7,55 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;600;700&family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a class="nav__list__link is-active" href="#">Apple</a>
+          </li>
+
+          <li>
+            <a class="nav__list__link" href="#">Samsung</a>
+          </li>
+
+          <li><a class="nav__list__link" href="#">Smartphones</a></li>
+
+          <li>
+            <a class="nav__list__link" data-qa="hover"
+          href="#"
+            >Laptops & Computers</a>
```

Comment:
<img width="1073" alt="Pasted Graphic" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/198bf6fa-1ac3-4a4e-9147-ce753a03cb3e">
fix it everywhere